sdk/python/kfp/v2/compiler/compiler.py

```diff
@@ -881,6 +876,9 @@ def _group_to_dag_spec(
                         raw_values, sort_keys=True)
                     subgroup_task_spec.parameter_iterator.item_input = (
                         input_parameter_name)
+                if (subgroup.parallelism > 0):
```

Comment:
I was hoping we could keep it here. Removing this line would actually cause all the IR to end up with an empty "IteratorPolicy: {}" field, which I would like to avoid. Since "0" is considered a default value, and gets stripped away during conversion to JSON by the formatter.

Any suggestion?

Reply:
Thanks for pointing that out. Makes sense to me! Let's keep it as is.